Makefile

```diff
@@ -66,16 +71,15 @@ vet: vet.client vet.controller vet.sidecar ## Vet code
 test: .test.proto test.client test.controller test.sidecar ## Run tests including unit tests
 
 .PHONY: test-e2e
-test-e2e: # Run e2e tests
-	@echo "unimplemented placeholder"
+test-e2e: chainsaw # Run e2e tests
+	$(CHAINSAW) test --values ./test/e2e/values.yaml
```

Comment:
Is my read correct that this expects `make cluster` and `make deploy` to have already been run? Or, another question: does this implicitly require ctlptl/kind?

Reply:
I expect that most users will be running `make cluster` and `make deploy`, but those should stay as separate targets. I see no point in trying to redeploy all components or reconfigure the cluster, if I want to test small change to the test logic itself.

Additionally, I sometimes run tests on remote dev clusters in the cloud, to see if I am able to find some conflicts with other components. `make cluster` will not be helpful in this case.

Reply:
Having `make cluster`/`make deploy` separate sounds good to me too. I am mostly wanting to make sure I have the right understanding here.

It seems like it could be useful to add a short statement to the doc that says this expects a cluster to be running and a version of COSI controller to be deployed (assuming both those statements are true). That might help answer some questions other contributors might have in the future.

Reply:
Added.